### DISCUSSION TITLE
Mob Stacking???????? Maybe???????

### DISCUSSION THREAD

**Discussion by YTMrSnipa** [Zrips/CMI#3905] [Feature Requests]
Can CMI have like a basic mob stacking feature? And you can choose the max stack size for each mob and enable/disable this module in modules.yml?

**Comment by WolverStones** [comment]
it's useless to be able to do it cmi are plugins and free that work without a problem ...

**Comment by CloudeLecaw** [comment]
What he mean's there are allready a bunch of up to date mobstacker plugins that does this kind of job without any problem, so at the Moment there is no real high priority for adding this kind of stacker into cmi.

**Comment by YTMrSnipa** [comment]
Yea I mean but a TON of them are very bad, and a lot of people want to be able to set the stack limit for each mob, so i was thinking maybe a poll/discussion in which they put all their feature requests into the stacking plugin...
Im just saying a majority of the stacking plugins take a HUGE TPS%, so maybe CMI can fix this or something?
